addons/l10n_br/models/res_company.py

```diff
@@ -16,3 +16,8 @@ class ResCompany(models.Model):
     def _localization_use_documents(self):
         self.ensure_one()
         return self.account_fiscal_country_id.code == "BR" or super()._localization_use_documents()
+
+    def _compute_show_taxes(self):
+        super()._compute_show_taxes()
+        for record in self:
+            record.show_taxes = record.country_code != 'BR'
```

Comment:
This is not correct I think, if you have for example one US company and one BR company in a db, one will override the value that is set in the function of the other localization. Also, I believe it is better to use the `account_fiscal_country_id.code` rather than the `country_code`
```suggestion
        for record in self:
            if record.account_fiscal_country_id.code == BR:
                record.show_taxes = False
```